hr_timesheet_sheet_attendance/tests/test_attendance_by_day.py

```diff
@@ -0,0 +1,73 @@
+from odoo.addons.hr_timesheet_sheet_attendance.tests.\
+    hr_timesheet_sheet_test_cases import HrTimesheetTestCases
+from odoo import fields
+
+
+class TestAttendanceByDay(HrTimesheetTestCases):
```

Comment:
There is a lot of duplicated code, maybe you could try to write a generic method to which you can pass a list of intervals and the expected result